ttls/client.py

```diff
@@ -123,10 +124,11 @@ def __init__(
         self._token = None
         self._details: dict[str, str | int] = {}
         self._default_mode = "movie"
+        self._api_version = api_version
 
     @property
     def base(self) -> str:
-        return f"http://{self.host}/xled/v1"
+        return f"http://{self.host}/xled/v{self._api_version}"
```

Comment:
If `self._api_version` is None this will fail, please raise ValueError in `base` if `self._api_version` is not set.